config/src/apps/vespa-get-config/getconfig.cpp

```diff
@@ -119,7 +119,7 @@ GetConfig::Main()
 
     const char *optArg = NULL;
     int optInd = 0;
-    while ((c = GetOpt("a:n:v:i:jm:c:t:V:w:r:s:p:dh", optArg, optInd)) != -1) {
+    while ((c = GetOpt("a:n:v:i:jlm:c:t:V:w:r:s:p:dh", optArg, optInd)) != -1) {
```

Comment:
this looks wrong, now 'j' will require and argument.  should be ..."i:jlm:"...

Reply:
Yeah, I saw and fixed that some minutes ago